docs/ipdk-dpdk.md

```diff
@@ -0,0 +1,123 @@
+# Building IPDK networking recipe for DPDK target
+
+## Overview
+
+This document describes how to build and run IPDK networking recipe on DPDK target.
+
+## Setup
+
+### Build P4 SDE for DPDK
+```bash
+git clone --recursive git@github.com:p4lang/p4-dpdk-target.git p4sde
+```
+
+For build instructions, refer [P4 SDE Readme](https://github.com/p4lang/p4-dpdk-target/blob/main/README.md#building-and-installing)
+
+### Build and install infrap4d dependencies
+
+```bash
+git clone --recursive git@github.com:ipdk-io/networking-recipe.git ipdk.recipe
+cd ipdk.recipe
+export IPDK_RECIPE=`pwd`
+cd $IPDK_RECIPE/setup
+cmake . -DCMAKE_INSTALL_PREFIX=`path for installing dependencies`
```

Comment:
This will do an in-source build. You generally want to do one of the following:

```cmake
cmake -B build -DCMAKE_INSTALL_PREFIX=<install path> [-DUSE_SUDO=ON]
cmake --build build [-j<njobs>]
```
or:
```cmake
mkdir build && cd build
cmake .. -DCMAKE_INSTALL_PREFIX=<install prefix> [-DUSE_SUDO=ON]
make [-j<njobs>]
```
The USE_SUDO option instructs CMake to build the dependencies in user mode and install them as root. Use this if you want to install in a directory such as `/usr/local` or `/opt/deps`.

> Google recommends that the dependency libraries be installed in a user directory, rather than a system directory such as `/usr` or `/usr/local`. An `/opt` subdirectory may be a useful compromise.